charts/rancher-logging/0.3.0/charts/log-aggregator/charts/log-aggregator-windows/values.yaml

```diff
@@ -5,7 +5,7 @@ labels: {}
 ##
 image:
   repository: rancher/log-aggregator
-  tag: v0.1.7
+  tag: v0.1.8
 
 nodeSelector: {}
 
```

Comment:
Can't comment below. Should questions.yaml be starting at 2.6.0 ? I guess it doesn't need to, since we have just left logging v1 open all the way, and <2.6 can't get to it.

Reply:
ill bump it anyway, it won't hurt